### DISCUSSION TITLE
节点的“mousewheel”事件监听不生效

### DISCUSSION THREAD

**Discussion by zhangyingnan001** [antvis/X6#4408] [Q&A]
x6事件定义中https://x6.antv.antgroup.com/tutorial/basic/events

鼠标滚轮 | cell:mousewheel | node:mousewheel | - | edge:mousewheel | blank:mousewheel

这些“mousewheel”事件监听不生效，请问怎样监听鼠标滚轮事件

**Comment by zlecheng** [comment]
> x6事件定义中https://x6.antv.antgroup.com/tutorial/basic/events
> 
> 鼠标滚轮 | cell:mousewheel | node:mousewheel | - | edge:mousewheel | blank:mousewheel
> 
> 这些“mousewheel”事件监听不生效，请问怎样监听鼠标滚轮事件

你是不是和画布的缩放冲突了，我的是可以监听到的

```js
  graph.on('node:mousewheel', ({ node }) => {
    console.log('鼠标滚轮缩放了');
  });
```
